PWGCF/JCorran/Core/JFFlucAnalysis.cxx

```diff
@@ -41,7 +43,8 @@ JFFlucAnalysis::JFFlucAnalysis(const char* /*name*/) : fVertex(0),
 }
 
 //________________________________________________________________________
-JFFlucAnalysis::JFFlucAnalysis(const JFFlucAnalysis& a) : fVertex(a.fVertex),
+JFFlucAnalysis::JFFlucAnalysis(const JFFlucAnalysis& a) : TNamed(),
```

Comment:
Although it seems they are not using the object name, in principle this should be 
```
JFFlucAnalysis::JFFlucAnalysis(const JFFlucAnalysis& a) : TNamed(a),
```

Reply:
Done